pkg/capabilities/consensus/ocr3/transmitter.go

```diff
@@ -0,0 +1,83 @@
+package ocr3
+
+import (
+	"context"
+
+	"github.com/smartcontractkit/libocr/offchainreporting2/types"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	pbtypes "github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3/types"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/core"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+type ContractTransmitter struct {
+	lggr        logger.Logger
+	registry    core.CapabilitiesRegistry
+	capability  capabilities.CallbackCapability
+	fromAccount string
+}
+
+func (c *ContractTransmitter) Transmit(ctx context.Context, configDigest types.ConfigDigest, seqNr uint64, rwi ocr3types.ReportWithInfo[[]byte], signatures []types.AttributedOnchainSignature) error {
```

Comment:
this new some commentary that explains how it's coupled via-spooky-action-at-a-distance to the capability `Execute`